docs/MAINTAINING.md

```diff
@@ -0,0 +1,43 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+# Maintaining
+
+- [Maintaining](#maintaining)
+  - [Workflow](#workflow)
+    - [Milestones](#milestones)
+    - [Project Boards](#project-boards)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+# Maintaining
+
+Current maintainers:
+
+- @code-asher
+- @oxy
+- @jsjoeio
+
+This document is meant to serve current and future maintainers of code-server, but also share openly our workflow for maintaining the project.
+
+## Workflow
+
+The workflow used by code-server maintainers is one that aims to be easy to understood by the community and easy enough for new maintainers to jump in and start contributing on day one.
+
+### Milestones
+
+We operate mainly using [Milestones](https://github.com/cdr/code-server/milestones). This was heavily inspired by our friends over at [vscode](https://github.com/microsoft/vscode).
+
+Here are the Milestones we use and how we use them:
+
+- "Backlog" -> Work not yet planned for a specific release.
+- "On Deck" -> Work under consideration for upcoming milestones.
+- "Backlog Candidates" -> Work that is not yet accepted for the Backlog. We wait for the community to weigh in.
+- "<Month> <Year>" -> Work to be done that month.
+
+### Project Boards
+
+We use project boards sparingly.
```

Comment:
I think one way we could describe the use of project boards is something like "for projects or goals that span multiple milestones". So things like testing, clean up, etc fall under that description.

Reply:
Good call!